examples/Console/TestPrometheusExporter.cs

```diff
@@ -42,9 +46,37 @@ internal static object Run(int port, int pushIntervalInSecs, int totalDurationIn
                 static_configs:
                 - targets: ['localhost:9184']
             */
-            System.Console.WriteLine("Press Enter key to exit.");
-            System.Console.ReadLine();
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddSource("TestMeter")
+                .AddPrometheusExporter(opt => opt.Url = $"http://localhost:{port}/metrics/")
```

Comment:
I like the simplicity here. User is not expected to anything about temporality (delta vs cumulative)!. Hope it can stay that way..

Reply:
Seems reasonable, because Prometheus is always cumulative right? I guess the only thing that would change this is if Prometheus introduced support for delta aggregations.

Reply:
yea.. Views will likely make this more complex :)